config/systems/todi.py

```diff
@@ -76,6 +76,46 @@
                 ],
             'launcher': 'srun',
         },
+        {
+            'name': 'debug',
+            'descr': 'GH200',
+            'scheduler': 'squeue',
+            'time_limit': '10m',
+            'container_platforms': [],
+            'access': ['--partition=debug'],
```

Comment:
Needs to be removed?

Reply:
Yes, I would say remove it for the moment and we can add it later in case we need.